modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -1,49 +1,51 @@
-//*SELinux policy differences. What are they?
-* In CL, metrics/telemetry data was collected by the update mechanism. In FCOS, the plan is to collect data (without unique identifiers) via https://github.com/coreos/fedora-coreos-pinger/issues/30[`fedora-coreos-pinger`]. Users will have the ability to disable data collection.
+* CL had three release channels: Stable, Beta, and Alpha. The FCOS production https://github.com/coreos/fedora-coreos-tracker/blob/master/Design.md#release-streams[release streams] are `testing`, `stable`, and `next`.
+* Automatic updates are now coordinated by Zincati, as described in the https://github.com/coreos/zincati/blob/master/docs/usage/auto-updates.md[Zincati documentation]. The rollback mechanism (via grub) is now provided by https://github.com/coreos/rpm-ostree/blob/master/README.md[`rpm-ostree`].
+* The `update-ssh-keys` tool is not shipped on FCOS. To update ssh keys, edit `authorized_keys` directly. Keep in mind that Ignition and Afterburn have their own files in `~/.ssh/authorized_keys.d`.
+* In general, SELinux confinement should work the same as in Fedora.
+* In CL, metrics/telemetry data was collected by the update mechanism. In FCOS, the data is collected (without unique identifiers) by https://github.com/coreos/fedora-coreos-pinger/issues/30[`fedora-coreos-pinger`]. You can disable data collection.
 * Cloud CLI clients are not included in FCOS. There is an initiative to create a "tools" container to run on FCOS.
-\\* Sticky directory - https://github.com/coreos/bugs/issues/2577
+//* Sticky directory - https://github.com/coreos/bugs/issues/2577
 * CL left Simultaneous Multi-Threading (SMT) enabled but advised users to turn it off if their systems were vulnerable to certain issues such as https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/l1tf.html[L1TF] or https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/mds.html[MDS]. By default, FCOS https://github.com/coreos/fedora-coreos-tracker/issues/181[automatically disables SMT] for vulnerable systems.
 
 == Implementation notes
 //* Partition layout differences. CL is at https://coreos.com/os/docs/latest/sdk-disk-partitions.html. I can't make heads or tails of the results of the discussions in https://github.com/coreos/fedora-coreos-tracker/issues/94.
-* The default filesystem choices on CL were ext4, btrfs, and xfs. On FCOS, the choices are xfs and ext4. If you wish to use btrfs, you can use xref:ign-storage.adoc[Ignition] to reformat `/var` to btrfs.
-//* sshd socket activation - I need more info on FCOS implementation. CL is here: https://coreos.com/os/docs/latest/customizing-sshd.html
-//* OEM partition support. Need more info.
+* The filesystem choices on CL were ext4 (default), btrfs, or xfs. On FCOS, you can use xfs (default) or ext4. If you wish to use btrfs, you can use xref:ign-storage.adoc[Ignition] to reformat `/var` to btrfs.
```

Comment:
Sorry, I forgot to comment on this before: the "filesystem choices" aren't any different on FCOS (AFAIK).  Only the defaults have changed.

Reply:
https://coreos.com/os/docs/latest/configuration.html states that btrfs was an option on CL.

Reply:
It should work on FCOS as well.

Reply:
I misunderstood the result of the discussion. I am rephrasing this to only note the change in default filesystem.